tests/e2e/vsan_stretched_cluster_utils.go

```diff
@@ -203,11 +203,17 @@ func waitForHostToBeDown(ip string) error {
 }
 
 // waitForAllNodes2BeReady checks whether all registered nodes are ready and all required Pods are running on them.
-func waitForAllNodes2BeReady(ctx context.Context, c clientset.Interface, timeout time.Duration) error {
-	framework.Logf("Waiting up to %v for all nodes to be ready", timeout)
+func waitForAllNodes2BeReady(ctx context.Context, c clientset.Interface, timeout ...time.Duration) error {
+	var pollTime time.Duration
+	if len(timeout) > 0 {
+		pollTime = timeout[0]
+	} else {
+		pollTime = pollTimeout * 5
```

Comment:
I meant to add an environment variable just like FULL_SYNC_WAIT_TIME for this, which we can set in the jobs and change as we need instead of the code change everytime

Reply:
code here looks fine
in case the new env variable is set, I want to use that timeout instead of the default one

Reply:
Added an env variable